src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from "react";
+import Image from "next/image";
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import Text from "src/components/Text";
+import * as S from "./styles";
+
+const ProfileContent: React.FC<Record<string, never>> = () => {
+    const [name, setName] = useState<string | null>(null);
+    const [subject, setSubject] = useState<string | null>(null);
+    const [content, setContent] = useState<string | null>(null);
+
+    const handleSubmit = (event: React.MouseEvent) => {
+      event.preventDefault();
+
+      let openerSubject: string = "PINUS Profile AMA: ";
+
+      if (name === null || name === "") {
+        setName("Anonymous");
+      }
+
+      if (subject === null || name === "") {
+        setSubject("Default");
+      }
+
+      if (content === null || name === "") {
+        setContent("Blank body");
+      }
+
+      if (!name && !subject && !content) {
+        return;
+      }
+
+      let finalSubject: string = openerSubject + subject;
+
+      const body =
+        `Hello,\n\n` +
+        `${content}\n\n` +
+        `What's up?.\n\n` +
+        `Best regards,\n\n` +
+        `${name}\n`;
+
+      location.href =
+        `mailto:albersutz123@gmail.com` +
```

Comment:
Hoho naughty copy from where ah. Anyways how would you change the location.href if I don't want to have cc?